packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -755,7 +755,7 @@ class ScrollableBaseNative extends Component {
 
 		this.lastInputType = 'pageKey';
 
-		this.scrollToAccumulatedTarget(pageDistance, canScrollVertically, this.props.overscrollEffectOn.pageKey);
+		this.scrollToAccumulatedTarget(pageDistance, canScrollVertically, this.props.overscrollEffectOn.pageKey, this.props.animate);
```

Comment:
I saw animation while paging feature validation. I pass `animate` here to disable animation explicitly.